src/coreclr/src/vm/loaderallocator.hpp

```diff
@@ -151,6 +151,8 @@ class LoaderAllocator
     // Heaps for allocating data that persists for the life of the AppDomain
     // Objects that are allocated frequently should be allocated into the HighFreq heap for
     // better page management
+
+    DAC_ALIGNAS(8) // Align the first member to an 8 byte boundary.  Windows does this by default, force Linux to match.
```

Comment:
Is this correct even on 32-bit platforms?

Reply:
This was added because with the x86 Windows compiler this member was aligned to a 8 byte boundary.  Based on all the diff I looked at, I am inferring that the first member is aligned to the `alignof` the class.  `DAC_ALIGNAS(LoaderAllocator)` would be semantically correct, but I am not sure if it is supported (because the type is incomplete when I need to write the line).

Reply:
On Windows, the class and first member is aligned to 8-bytes even on x86/arm because
- The `uint64_t m_nLoaderAllocator` member
- Windows packing is required to be set to 8. Linux must be configured to match.

Reply:
Can this be `DAC_ALIGNAS(m_nLoaderAllocator)`? Or at lest refer to `m_nLoaderAllocator` in the comment?